README.md

```diff
@@ -48,7 +48,6 @@ packages/
   okta-angular
   okta-react
   okta-react-native
-  okta-vue
```

Comment:
I think we should include a link to the new home for this repo.

Otherwise I'm entirely giddy about this.